modules/nw-sriov-example-dpdk-line-rate.adoc

```diff
@@ -0,0 +1,278 @@
+// Module included in the following assemblies:
+//
+// * networking/hardware_networks/using-dpdk-and-rdma.adoc
+
+:_content-type: PROCEDURE
+[id="nw-example-dpdk-line-rate_{context}"]
+= Using SR-IOV and the Node Tuning Operator to achieve a DPDK line rate
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Install the SR-IOV Network Operator.
+* Log in as a user with `cluster-admin` privileges.
+* Deploy a standalone Node Tuning Operator.
+
+The Node Tuning Operator is used to configure isolated CPUs, hugepages, and a topology scheduler.
+You can then use the Node Tuning Operator with SR-IOV to achieve a specific DPDK line rate networking.
+
+[NOTE]
+====
+In previous versions of OpenShift Container Platform, the Performance Addon Operator was used to implement automatic tuning to achieve low latency performance for OpenShift applications. In OpenShift Container Platform 4.11, this functionality is part of the Node Tuning Operator.
```

Comment:
Please replace each instance of "OpenShift Container Platform" with "`{product-tite}`".

You could also replace "4.11" with "`{product-version}`".